core/state/state_object.go

```diff
@@ -157,13 +157,11 @@ func (s *stateObject) getTrie(db Database) (Trie, error) {
 			// prefetcher
 			s.trie = s.db.prefetcher.trie(s.addrHash, s.data.Root)
 		}
-		if s.trie == nil {
```

Comment:
s.trie gets modified 2 lines above

Reply:
any reason to write the if `s.trie == nil {}` twice?

Reply:
my bad